### PR TITLE
Additional updates to PRR docs

### PR DESCRIPTION
**Pull request by bridgetkromhout**
Updates based on feedback.

### CODE REVIEW

File: keps/sig-network/563-dual-stack/README.md [hunk 1]
```diff
@@ -1540,15 +1546,22 @@ resource usage (CPU, RAM, disk, IO, ...) in any components?**
...
 
 * **What are other known failure modes?**
-  
+
+  * Missing prerequisites. Operator must verify the following conditions:
+    1. Ensure correct support in the node infrastructure provider.
+      a. supports routing both IPv4 and IPv6 interfaces.
+      b. makes both IPv4 and IPv6 interfaces available to Kubernetes.
+    2. CNI needs to be correctly configured for dual-stack service.
+      a. Pods need to be able to use dual-stack service.
```
Comment (lachie83) [inline keps/sig-network/563-dual-stack/README.md]:
```suggestion
    2. CNI provider needs to be correctly configured for dual-stack.
      a. Kubernetes must be able to assign IPv4 and IPv6 IP addresses from the CNI provider.
```

File: keps/sig-network/563-dual-stack/README.md [hunk 2]
```diff
@@ -1540,15 +1546,22 @@ resource usage (CPU, RAM, disk, IO, ...) in any components?**
...
+    3. Service cidrblock for controller manager must be available.
```
Comment (lachie83) [inline keps/sig-network/563-dual-stack/README.md]:
```suggestion
    3. Dual-stack CLI flags must be configured on the cluster as defined in the [docs](https://kubernetes.io/docs/concepts/services-networking/dual-stack/#enable-ipv4-ipv6-dual-stack)
```

File: keps/sig-network/563-dual-stack/README.md [hunk 3]
```diff
@@ -1392,26 +1392,35 @@ This capability will move to stable when the following criteria have been met.
...
-  services are converted to single-stack first (switch ipfamily to single-stack
-  on all services) and then disable the feature. Remove the CLI parameters, as
-  an older client won't see or be able to use the new fields. When the user
-  disables dual-stack from the controller manager, new endpoints will no longer
-  carry two sets, while existing endpoints may not be updated.
+  Yes. If you decide to turn off dual-stack after turning on:
+    1. Ensure all services are converted to single-stack first (downgraded to
+       single-stack as described in this KEP)
+    2. Remove the CLI parameters.
+    3. Disable the feature.
```
Comment (lachie83) [inline keps/sig-network/563-dual-stack/README.md:1405]: Are these two discrete steps?

File: keps/sig-network/563-dual-stack/README.md [hunk 4]
```diff
@@ -1560,25 +1583,33 @@ resource usage (CPU, RAM, disk, IO, ...) in any components?**
...
+    2. If a workload doesn't account for being offered both IP families, it
+       may fail in unexpected ways. For example, firewall rules may need to be
+       updated to allow IPv6 addresses.
+    3. Recommended to independently verify legacy workloads to ensure fidelity.
+
+  *  IP-related error conditions to consider.
+    1.  pod IP allocation fails (this is due to CNI)
+      a. Will result in the pod not running if there is no IP allocated from
+         the CIDR.
+    2. routing fails
```
Comment (lachie83) [inline keps/sig-network/563-dual-stack/README.md]:
```suggestion
    2. Service to pod routing fails
```

File: keps/sig-network/563-dual-stack/README.md (hunk 3 above)
Comment (bridgetkromhout) [reply keps/sig-network/563-dual-stack/README.md:1405]: Yes; Kal stipulated that they should be in that order.

**Pull request merged**